front_end/server/lib/routes/servers.js

```diff
@@ -1,4 +1,19 @@
 const express = require('express');
 const router = express.Router();
 const Server = require('../models/server');
-const bodyParser = require('body-parser').json();
\ No newline at end of file
+const bodyParser = require('body-parser').json();
+
+router
+  .get('/', (req, res, next) => {
+    Server.find()
+      .lean()
+      .then(servers => res.send(servers))
+      .catch(next);
+  })
+  .post('/', bodyParser, (req, res, next) => {
+    new Server(req.body).save()
```

Comment:
Hahah yes, "newServer" could be called anything -- it's just what we're naming the param for what gets spit out of the save() method. In this case, I'm trying to emphasize "this is a new Server-schema-based object that was stored in our DB"